src/app/features/ledger/flows/message-signing/use-message-type.ts

```diff
@@ -1,11 +1,11 @@
 import { ClarityValue } from '@stacks/transactions';
 
-import { SignedMessage, StructuredMessageDataDomain } from '@shared/signature/signature-types';
+import { StructuredMessageDataDomain, UnsignedMessage } from '@shared/signature/signature-types';
 import { isString } from '@shared/utils';
 
 import { useLocationStateWithCache } from '@app/common/hooks/use-location-state';
 
-export function useSignedMessageType(): SignedMessage | null {
+export function useSignedMessageType(): UnsignedMessage | null {
```

Comment:
Should this be `useUnsignedMessageType` if you are now passing in `UnsignedMessage`?